<?xml version="1.0"?>

<launch>
  <node pkg="mcl_gpu" type="mcl_gpu" name="mcl_gpu" output="screen">
    <param name="which_rm" value="rmgpu"/>
    <param name="scan_topic" value="scan"/>
    <param name="max_range" value="30"/>
  </node>
</launch>
